dwave/preprocessing/include/dwave/presolve.hpp

```diff
@@ -182,8 +194,34 @@ class Presolver {
         }
     }
 
-    // todo: break into separate presolver
-    void substitute_self_loops() {
+    //----- One-time Techniques -----//
+
+    void technq_spin_to_binary() {
```

Comment:
IMO if we want to prepend all of these with "technique", we might as well prepend the entire word. Only two more characters